pandas/tests/tslibs/test_conversion.py

```diff
@@ -139,3 +143,15 @@ def test_localize_pydatetime_dt_types(dt, expected):
     # localize_pydatetime
     result = conversion.localize_pydatetime(dt, UTC)
     assert result == expected
+
+
+@pytest.mark.xfail(
+    not PY39,
+    reason="ZoneInfo objects were introduced in Python 3.9",
+)
+def test_zoneinfo_support():
+    # GH 37654
+    # Ensure that TimeStamp supports ZoneInfo objects
+    tz = ZoneInfo("US/Pacific")
```

Comment:
As a suggestion, the `tz_aware_fixture` object should include a `ZoneInfo` object so it's exposed across the testing suite.

P.S. Glad to hear you're doing well

Reply:
@mroeschke Thank you! I'm very happy to feel better too.

Created the fixture, so that should make my future breaking the tz conversion and `Timestamp` much easier to debug. 

Maybe you have any idea why this new test could be throwing an unraisable error, though? I'm a bit stumped, because in all other tests (or if I just open the console and execute the code), the error propagates through the call stack properly. I'll probably replace the test or cludge something together tomorrow, but I don't want to accumulate tech debt in case it's something less innocuous than it looks to be:

```python
pandas/tests/tslibs/test_conversion.py::test_zoneinfo_support
  C:\Users\amatamune\Anaconda3\envs\pandas-dev\lib\site-packages\_pytest\unraisableexception.py:78: PytestUnraisableExceptionWarning: Exception ignored in: 'pandas._libs.tslibs.tzconversion.tz_convert_from_utc_single'

  Traceback (most recent call last):
    File "pandas\_libs\tslibs\timezones.pyx", line 263, in pandas._libs.tslibs.timezones.get_dst_info
      num = int(get_utcoffset(tz, None).total_seconds()) * 1_000_000_000
  AttributeError: 'NoneType' object has no attribute 'total_seconds'

    warnings.warn(pytest.PytestUnraisableExceptionWarning(msg))
```

Reply:
Got it, thank you, that's super-useful!

Reply:
We probably won't need that test anyway. Adding zoneinfo to fixture seems to take care of testing.